qa-tests-backend/scripts/run-compatibility.sh

```diff
@@ -119,4 +124,34 @@ shorten_tag() {
     fi
 }
 
+verify_scannerV2_deployed() {
+    local namespace=${1:-stackrox}
+    info "Waiting for Scanner V2 deployment to appear in namespace ${namespace}..."
+    wait_for_object_to_appear "$namespace" deploy/scanner-db 600
+    wait_for_object_to_appear "$namespace" deploy/scanner 300
+    info "** Scanner V2 is deployed in namespace ${namespace}"
+}
+
+verify_scannerV4_deployed() {
+    local namespace=${1:-stackrox}
+    verify_scannerV4_indexer_deployed "$namespace"
+    verify_scannerV4_matcher_deployed "$namespace"
+}
+
+verify_deployment_scannerV4_env_var_set() {
+    local namespace=${1:-stackrox}
+    local deployment=${2:-central}
+    local deployment_env_vars
+    local scanner_v4_value
+
+    deployment_env_vars="$("${ORCH_CMD}" </dev/null -n "${namespace}" get deploy/"${deployment}" -o jsonpath="{.spec.template.spec.containers[?(@.name=='${deployment}')].env}")"
+    scanner_v4_value="$(echo "${deployment_env_vars}" | jq -r '.[] | select(.name == "ROX_SCANNER_V4").value')"
+
+    if [[ "${scanner_v4_value}" == "true" ]]; then
+        return 0
+    else
+        return 1
+    fi
```

Comment:
That can be simply written as:
```suggestion
    [[ "${scanner_v4_value}" == "true" ]]
```

Reply:
You're right, my bad. I copied this function over from our scanner tests without looking at it closely enough.